CompositeUi/src/views/page/organization/AddMemberToOrganization.js

```diff
@@ -0,0 +1,80 @@
+/*
+ * This file is part of the Kreta package.
+ *
+ * (c) Beñat Espiña <benatespina@gmail.com>
+ * (c) Gorka Laucirica <gorka.lauzirika@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+import './../../../scss/views/page/project/_settings';
+
+import React from 'react';
+import Search from './../../component/SearchMember';
+import {connect} from 'react-redux';
+import MemberActions from './../../../actions/Member';
+import UserPreview from './../../component/UserPreview';
+import Button from './../../component/Button';
+
+@connect(state => ({currentOrganization: state.currentOrganization}))
+class AddMemberToOrganization extends React.Component {
+  static propTypes = {
+    onMemberRemoveClicked: React.PropTypes.func,
+    organization: React.PropTypes.object
+  };
+
+  filterMembersToAdd(q) {
+    this.props.dispatch(
+      MemberActions.fetchMembersToAdd(
+        q,
+        this.props.currentOrganization.organization.organization_members.map((item) => item.id)
+      )
+    );
+  }
+
+  triggerOnMemberaddClicked(participant) {
+    this.props.onMemberAddClicked(participant);
+  }
+
+  renderMembersToAdd() {
+    return this.props.currentOrganization.potential_members
+      .map((member, index) => {
+        const actions = (
+          <Button color="green"
+              onClick={this.triggerOnMemberaddClicked.bind(this, member)}
+              type="icon">
+              <i className="fa fa-plus"/>
```

Comment:
Font awesome dependency is previously removed so this css clases do not have any effect